### ISSUE TITLE
Replace rollup-plugin-json with @rollup/plugin-json

### ISSUE THREAD

**Issue by gavinr** [Esri/esri-leaflet-geocoder#248]
When running `npm install`, I get an error:

```
[!] Error: Cannot find module '@rollup/plugin-json'
Require stack:
- ....\esri-leaflet-geocoder\profiles\debug.js
- ....\esri-leaflet-geocoder\node_modules\rollup\bin\rollup
Error: Cannot find module '@rollup/plugin-json'
Require stack:
- ....\esri-leaflet-geocoder\profiles\debug.js
- ....\esri-leaflet-geocoder\node_modules\rollup\bin\rollup
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (....\esri-leaflet-geocoder\profiles\debug.js:5:28)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.require.extensions..js (....\esri-leaflet-geocoder\node_modules\rollup\bin\rollup:2799:19)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
```

**Comment by jgravois** [comment]
the perils of relying on the build profile in esri-leaflet itself. 😄 
https://github.com/Esri/esri-leaflet-geocoder/blob/059a831609a239cf568a31cbff8d43028e1a0e3b/profiles/base.js#L1

**Comment by gavinr** [comment]
Ah, so you're saying since we're relying on that, and we just switched `esri-leaflet` to the newer plugin (https://github.com/Esri/esri-leaflet/commit/953411223d5fb4c5630c864f23e5ed58563ba2fb) we'll need to make similar updates here. 👍

**Comment by jgravois** [comment]
yeah.

sometimes having a DRY rollup setup saves us from making the same change in more than one plugin, other times it has the opposite effect. 😬

**Comment by jwasilgeo** [comment]
Closed via PR #249.

**Comment by gavinr** [comment]
This was released in [v2.3.3](https://github.com/Esri/esri-leaflet-geocoder/releases/tag/v2.3.3)
